Plugins/Public/mobiledocking_plugin/Main.cpp

```diff
@@ -194,6 +229,9 @@ void __stdcall BaseExit(uint iBaseID, uint iClientID)
 
 }
 
+// Temporary storage for client data to be handled in LaunchPosHook.
+CLIENT_DATA undockingShip;

```

Comment:
note to self: this is done in the base plugin already. Set in PlayerLaunch, read in LaunchPosHook